src/mpl/src/MacroPlacer.cpp

```diff
@@ -472,14 +472,14 @@ void MacroPlacer::placeMacrosCornerMaxWl()
   }
 
   if (found_best) {
-    logger_->info(MPL, 73, "Best weighted wire length {:g}", bestWwl);
+    logger_->info(MPL, 73, "Best weighted wire length {:g}.", bestWwl);
     std::vector<Partition> best_set = allSets[best_setIdx];
     for (auto& best_partition : best_set) {
       updateMacroLocations(best_partition);
     }
     updateDbInstLocations();
   } else
-    logger_->warn(MPL, 72, "No partition solutions found.");
+    logger_->warn(MPL, 72, "No partition solutions found..");
```

Comment:
There are two periods and the end of the sentence

Reply:
Fixed on 7d3b4ed